randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -163,11 +170,23 @@ Extra - asset_id: 4
   * Layers: default
   * Normal Door to Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
-      Grapple Beam or Movement (Beginner)
+      Any of the following:
+          Grapple Beam or Space Jump
+          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Movement (Intermediate)
+          All of the following:
+              # From the sea floor
+              Gravity Suit
+              Any of the following:
+                  Hi-Jump Boots or Wall Jumps (Beginner)
+                  Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
   > Pickup (Seafloor Missile Expansion)
       All of the following:
           Morph Ball
-          Gravity Suit or Knowledge (Intermediate)
+          Any of the following:
+              Morph Ball Bombs
```

Comment:
This should be "can lay any bombs" as PBs also work

Reply:
Updated to anybombs